nncf/openvino/hardware/fused_patterns.py

```diff
@@ -575,6 +575,18 @@ def create_linear_arithmetic_activations() -> GraphPattern:
     return linear
 
 
+@OPENVINO_HW_FUSED_PATTERNS.register(HWFusedPatternNames.LINEAR_ARITHMETIC_ACTIVATIONS_ARITHMETIC)
+def create_linear_arithmetic_activations() -> GraphPattern:
```

Comment:
The same comment about pattern reuse and the method's name.
Also, it would be useful to add a comment about the model where this pattern can be observed since we're not adding a test for it.

Reply:
I added comment in the `patterns.py`